testdata/script/literals.txtar

```diff
@@ -330,7 +332,66 @@ func dotImportTest() {
     println(DotImportedFunc())
     println(DotImportedVar)
     println(DotImportedType("str as dot imported type"))
+    println(DotImportedStruct.Str)
 }
+
+func multipleTimeImportTest() {
+    regularAndUnusedRenamed()
+    regularAndUnusedDotImport()
+    unusedRegularAndDotImport()
+}
+
+func multipleCheck(vals ...string) {
+	if len(vals) < 2 {
+		return
+	}
+	for i := 0; i < len(vals)-1; i++ {
+		if vals[i] != vals[i+1] {
+			panic("vals must be equals")
+		}
+	}
+}
+
+-- regular_and_unused_renamed.go --
+package main
+
+import (
+    "test/main/imp_mult"
+    imp_mult2 "test/main/imp_mult"
+)
+
+func regularAndUnusedRenamed() {
+    const localConst = imp_mult.MultImpStr
+    multipleCheck(imp_mult.MultImpStr, imp_mult2.MultImpStr)
+}
+
+-- regular_and_unused_dotimport.go --
+package main
+
+import (
+    "test/main/imp_mult"
+    . "test/main/imp_mult"
+)
+
+func regularAndUnusedDotImport() {
+    const localConst = imp_mult.MultImpStr
```

Comment:
is this how you're keeping one of the imports used? it's a bit fragile, because we might end up "inlining" or simplifying all constant declarations in the future as part of obfuscation. it's probably better to have a real use via a func call.

Reply:
const is specially chosen because if i pass it as a parameter, then literals will just inline it and obfuscate it :)